docs/notebooks/extreme_value_analysis.ipynb

```diff
@@ -22,7 +22,7 @@
   },
```

Comment:
<p>We can already inspect the Dataset, so I don't think that these new cells are necessary (here and similar ones elsewhere). But no strong opinion either way.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/hydrologie/xhydro/pull/283/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>